### PR TITLE
Add extends for calibre light

### PR DESCRIPTION
**Pull request by ckeisato**
Remove all font extends

### CODE REVIEW

**Comment by josephco** [comment]
This looks like a fine solution to me. Let's see what the others think.

**Comment by josephco** [comment]
This might be a good opportunity to convert our font extends to mixins, the code is starting to feel a little unwieldy. If we need to re-declare Calibre Light for all of these font definitions, let's leave a @todo to remove them later once we are fully integrated.

File: src/forms/_base.scss [hunk 1]
```diff
@@ -228,9 +228,8 @@ select {
 
 
 .form-label {
-  @extend %headers;
-
-  @extend %h4;
+  @include headers;
```
Comment (josephco) [inline src/forms/_base.scss]: Can probably delete this line, but test it.

**Comment by josephco** [comment]
LGTM, DTM but maybe get @robwierzbowski or @stewartyu to give it a once-over.

**Comment by robwierzbowski** [comment]
Nice. The compiler should catch any missing extend calls, and the diff looks great. DTM.

**Comment by stewartyu** [comment]
Sorry, I was looking at this earlier today but got sidetracked. LGTM!

**Pull request merged**